data/blog/eleventh-wave-of-nostr-grants.mdx

```diff
@@ -0,0 +1,173 @@
+
+Secure Direct Messages brings end-to-end encrypted private messaging to nostr
+clients by offering a library that implements the Double Ratchet algorithm. By
+protecting message metadata and securing histories even if a user's nostr key is
+compromised, this approach significantly enhances privacy across the network.
+The protocol is currently available as a TypeScript library and integrated into
+clients such as [Iris](https://iris.to/) and [Damus](https://damus.io/).
+Complementary work includes a high-performance
+[Rust/LMDB](https://crates.io/crates/nostr-lmdb) server for notifications and
+webhook subscriptions.
+
+Upcoming milestones include group messaging support, multi-device session
+management [modeled on Signal](https://signal.org/docs/specifications/sesame/),
+and the public release of Double Ratchet in Damus. The team also plans to
+finalize and standardize the protocol through [NIPs 117 and
+118](https://github.com/nostr-protocol/nips/pull/1813/commits), expanding
+integration across both web and native clients. This work aims to make secure,
+decentralized messaging a native capability across the nostr ecosystem.
+
+Repository:
+[mmalmi/nostr-double-ratchet](https://github.com/mmalmi/nostr-double-ratchet)  
+License: MIT
+
+### Nostr Game Engine
+
+[Nostr Game Engine](https://nge.rblb.it/) is building a free, open-source,
+royalty-free engine for game developers—powered by nostr. Built using
+[jMonkeyEngine](https://github.com/jMonkeyEngine/jmonkeyengine?tab=readme-ov-file#jmonkeyengine),
+the project replaces traditional centralized systems with nostr-native modules
+while preserving compatibility. The initial release consists of modular
+extensions, with plans to evolve into a dedicated engine with improved
+architecture.
+
+This grant will support development across several areas: interoperable player
+authorization via
+[NIP-46](https://github.com/nostr-protocol/nips/blob/master/46.md) signers,
+WebRTC signaling over nostr, in-game advertising via nostr-DAN, matchmaking,
+social profiles via
+[NIP-58](https://github.com/nostr-protocol/nips/blob/master/58.md), social
+features ([NIP-17](https://github.com/nostr-protocol/nips/blob/master/17.md) and
+[NIP-28](https://github.com/nostr-protocol/nips/blob/master/28.md)), DRM/release
+tools, and micropayments via NWC, Cashu, and Lightning. The goal is to offer a
+full-featured, cross-platform engine for browsers, desktops, and mobile that
+requires only a nostr relay as the backend—significantly reducing
+infrastructure burdens for independent creators.
+
+Repository: [github.com/riccardobl](http://github.com/riccardobl)  
+License: BSD-3-Clause
+
+---
+
+We're proud to support these projects and look forward to their continued
+contributions to the nostr ecosystem. OpenSats will continue funding free and
+open-source work that strengthens decentralization, privacy, and resilience. If
+you're building something aligned with these values, [we encourage you to
```

Comment:
> If you're building something aligned with these values, [we encourage you to apply](https://os-website-git-11th-wave-of-nostr-grants-opensats.vercel.app/apply).

...aligned with [our mission](https://opensats.org/mission), we encourage you to [apply for funding](https://opensats.org/apply).